app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -111,26 +109,15 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
             )
         }
 
-        if (Settings.getInstance(this@MainActivity).shouldShowTurboModeOnboarding()) {
-            val onboardingIntent = Intent(this@MainActivity, OnboardingActivity::class.java)
-            startActivityForResult(onboardingIntent, ONBOARDING_REQ_CODE)
+        if (Settings.getInstance(this@MainActivity).shouldShowPocketOnboarding()) {
```

Comment:
I'm not sure if this changes anything that matters, but it looks like we're changing the order of execution here (the Pocket onboarding block is now executed before the Turbo one).

I did some brief testing and everything _seems_ to be working as expected, so I'm not sure if that has any impact at all.

Reply:
Activities open in a stack so unless we want the pocket onboarding to show first, this is the correct way